docs/reference/setup/jvm-options.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[jvm-options]]
+=== Setting JVM options
+
+The preferred method of setting Java Virtual Machine options (including
+system properties and JVM flags) is via the `jvm.options` configuration
+file. The default location of this file is `config/jvm.options` (when
+installing from the tar or zip distributions) and
+`/etc/elasticsearch/jvm.options` (when installing from the Debian or RPM
+packages).
+
+This file contains a line-delimited list of JVM arguments following
+a special syntax:
+
+* lines consisting of whitespace only are ignored
+* lines beginning with `#` are treated as comments and are ignored
++
+[source,text]
+-------------------------------------
+# This is a comment
+-------------------------------------
+
+* lines beginning with a `-` are treated as a JVM option that applies
+  independent of the version of the JVM
++
+[source,text]
+-------------------------------------
+-Xmx2g
+-------------------------------------
+
+* lines beginning with a number followed by a `:` followed by a `-` are treated
+  as a JVM option that applies only if the version of the JVM matches the number
++
+[source,text]
+-------------------------------------
+8:-Xmx2g
+-------------------------------------
+
+* lines beginning with a number followed by a `-` followed by a `:` are treated
+  as a JVM option that applies only if the version of the JVM is greater than or
+  equal to the number
++
+[source,text]
+-------------------------------------
+8-:-Xmx2g
+-------------------------------------
+
+* lines beginning with a number followed by a `-` followed by a `:` followed by
+  a `-` followed by a number are treated as a JVM option that applies only if
+  the version of the JVM falls in the range of the two numbers
++
+[source,text]
+-------------------------------------
+8-:-9-Xmx2g
```

Comment:
Is this syntax really correct?  Shouldn't it just be `8-9:-Xmx2g`?

Reply:
Oops, you're right, the description is off.

Reply:
I pushed 09549c4f460363b3da2b3a3ea1b584a17aa8d714.